app/Models/Eventlog.php

```diff
@@ -59,6 +75,10 @@ public static function log($text, $device = null, $type = null, $severity = Aler
             'username'  => (class_exists('\Auth') && Auth::check()) ? Auth::user()->username : '',
         ]);
 
+        if (is_numeric($device)) {
```

Comment:
Hmm, I think I purposely required a Device model.  I let the event_log() function convert int to Device before calling Log::event()

Reply:
log_event probably does